src/env/test/mod.rs

```diff
@@ -98,6 +98,16 @@ impl TestEnv {
     pub fn rng(&mut self) -> &mut TestRng256 {
         &mut self.rng
     }
+
+    pub fn set_enterprise_attestation(&mut self) {
```

Comment:
I wonder if this is the right place to add this function. It only affects the storage, not customization, so it's not really something you do to env, but that uses env to affect storage.

We could end up with a lot of these helper functions that take an `&mut Env` for test purposes and do some setup with it. Maybe a helper mod might be appropriate, in anticipation?

Reply:
Yes, I had a similar concern that `env` should probably not be used for what can be done through the library API.

I don't think a helper mod is needed for tests, they already take `TestEnv`. But it could be used to organize testing code, for example `env::test::storage::enable_enterprise_attestation(&mut TestEnv)` instead of `env::test::TestEnv::enable_enterprise_attestation()`. I don't have strong opinion about that. For now I think we don't need it, but we might indeed need it in the future if we end up with more such functions. Ideally I would hope we actually don't need this function and can directly call into the library API.